crates/fluvio-sc-schema/Cargo.toml

```diff
@@ -24,9 +24,8 @@ paste = "1.0"
 
```

Comment:
This crate's version needs to update patch number as well

Reply:
Adressed on: [1ebd052](https://github.com/infinyon/fluvio/pull/2738/commits/1ebd052da8e2352d3f63875a0d6e6e76855421f8)